.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,30 @@
+### Release Summary:
+<!-- If this is a feature or bug that impacts customers and is significant enough to include in the "Summary" section of the next version release, please include a brief (1-2 sentences) description of the change. The audience of this summary is future customers, not maintainers or reviewers. See https://github.com/aws/s2n-tls/releases/tag/v1.5.7 for an example. Otherwise, leave this section blank -->
```

Comment:
Kind of weird ordering. Shouldn't this be at the end of the template?

Reply:
I'm imagining that I'm the one doing the release. I do NOT want to scroll to find that message when I'm already opening a million links :) Maybe once we've automated it we can move it down

Reply:
Oh eh, yeah from a releaser's perspective it's better to be on the top.